engine/src/main/java/org/terasology/rendering/dag/AbstractNode.java

```diff
@@ -115,4 +115,18 @@ public boolean isEnabled() {
     public void setEnabled(boolean enabled) {
         this.enabled = enabled;
     }
+
+    /**
+     * Utility method to conveniently retrieve materials from the Assets system,
+     * hiding the relative complexity of the exception handling.
+     *
+     * @param materialUrn a ResourceUrn instance providing the name of the material to be obtained.
+     * @return a Material instance
+     * @throws RuntimeException if the material couldn't be resolved through the asset system.
+     */
+    public static Material getMaterial(ResourceUrn materialUrn) {
+        String materialName = materialUrn.toString();
+        return Assets.getMaterial(materialName).orElseThrow(() ->
+                new RuntimeException("Failed to resolve required asset: '" + materialName + "'"));
+    }
```

Comment:
I feel a bit cheeky placing this method here. Almost all nodes however load a Material and it does simplify the code a little.